migrations/deployETO.js

```diff
@@ -266,7 +272,7 @@ export async function checkETO(artifacts, config, etoCommitmentAddress) {
   );
   const termsInUniverse = await universe.isInterfaceCollectionInstance(
     knownInterfaces.termsInterface,
-    await eto.etoTerms(),
+    await etoTerms.address,
```

Comment:
seems like we don't need to `await` here?

Reply:
yeah! will fix thx